src/rdbms/mongoose_rdbms.erl

```diff
@@ -734,6 +735,16 @@ db_engine(_HostType) ->
     try mongoose_backend:get_backend_name(global, ?MODULE)
     catch error:badarg -> undefined end.
 
+%% @doc Used to optimise queries for different databases.
+%% @todo Should be refactored to use host types with this module.
+%% Also, this parameter should not be global, but pool-name parameterized
+-spec db_type() -> mssql | generic.
+db_type() ->
+    case ejabberd_config:get_local_option(rdbms_server_type) of
+        "mssql" -> mssql;
```

Comment:
I think this will be returned as an atom, right? https://github.com/esl/MongooseIM/blob/d856ba0d333f7aac9c4c97a45614327c40de6079/src/config/mongoose_config_spec.erl#L213 I think the quotes before where due to some list-to-atom-with-flatten-to-list weird hack 🤔

Reply:
True, good catch. Will fix it.